apps/frontend/src/libs/components/link/link.tsx

```diff
@@ -5,11 +5,18 @@ import { type ValueOf } from "~/libs/types/types.js";
 
 type Properties = {
 	children: React.ReactNode;
+	className: string;
```

Comment:
Try not to pass `className` prop, instead add different props to a component to make it customizable